target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -96,6 +96,9 @@ ramips_setup_interfaces()
 	tplink,tl-wpa8631p-v3)
 		ucidef_set_interface_lan "lan1 lan2 lan3 plc0"
 		;;
+	tplink,mr600-v2-eu)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" "wan"
```

Comment:
You can configure the wwan0 port now too, see bf9dc176516bdf030486483e3b766b9f49f3db15

Reply:
Maybe it's also time to decide if we need a convenient wrapper for this?
https://github.com/openwrt/openwrt/pull/10632#issuecomment-1281503138
@Leo-PL

Reply:
@bam80 I'm in favor of this and wanted to do that, but had no time to implement it this weekend.

Reply:
@Leo-PL please ping me if you do, I'll update my device to utilize the wrapper

Reply:
> You can configure the wwan0 port now too, see [bf9dc17](https://github.com/openwrt/openwrt/commit/bf9dc176516bdf030486483e3b766b9f49f3db15) and #10972

I've just pushed a fixup for this, thanks for the heads up!